### DISCUSSION TITLE
What functions do you use?

### DISCUSSION THREAD

**Discussion by jcorporation** [jcorporation/myMPD#421] [General]
Some kind of survey:
Which myMPD functions do you use intensively? Which other functions are useless for you?

**Comment by Emilot** [comment]
Every function except these:

1) Jukebox
2) Home
3) Smart playlist
4) Crossfade

**Reply by john-** [reply]
I pretty much only use these functions:

1.  Jukebox
2. Volume
3. Skip  :)

**Comment by jcorporation** [comment]
I personally use jukebox in conjunction with smart playlists.

**Comment by jdr85** [comment]
I use:
- Playing music of course
- Basic functions like Next/Previous/Play/Pause, Volume
- Playback Screen with covers
- Queue management (save queue, shuffle queue, clear/crop queue, search, reorder with Drag&Drop) 
- Queue: Play after current playing song (I really love that you implemented my request!)
- Search
- Add to queue from filesystem
- Add to queue from m3u playlists
- Webradio stream (I also use the station artwork)
- System Commands (deprecated) / Lua for shutdown
- Sink selection (in the volume popup)

I don't use (not exhaustive):
- Homescreen
- Smart Playlists
- Jukebox

**Reply by Greylinux** [reply]
I use:

- search
- web radio stream
- timers ( although having the ability to set a specified amount of minutes rather than end time would be a great addition)
- volume
- playlists ( not smart playlists)

**Comment by therion23** [comment]
Whatever it takes to find what i feel like listening to, which mostly means the file system browser, the search function when that fails, and of course the queue screen when i searched for the wrong thing.

Play/Pause/Skip/usw i handle through Home Assistant, making a good playlist for the moment is what i use myMPD for.
